### PR TITLE
Added lambda function url to terraform

### LINKED ISSUE #12: Create a url endpoint accessible to the clients browsers

### PR DESCRIPTION
**Pull request by hhldiniz**
This adds an endpoint that can be accessed from the clients

### CODE REVIEW

**Comment by bezmaxwell** [comment]
Nice.It's working

File: .terraform/terraform.tfstate [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": 3,
+    "serial": 1,
+    "lineage": "abf406aa-985a-d96f-7477-f5a782d01696",
+    "backend": {
+        "type": "cloud",
+        "config": {
+            "hostname": null,
+            "organization": "hugodiniz",
+            "token": null,
```
Comment (bezmaxwell) [inline .terraform/terraform.tfstate:10]: After we can use this token for authentication in pipeline, right?

**Review by bezmaxwell — APPROVED** [review]
LGTM

File: .terraform/terraform.tfstate (hunk 1 above)
Comment (hhldiniz) [reply .terraform/terraform.tfstate:10]: This is only the terraform state file. There's a copy of this in the cloud, so I believe I can remove this sometime soon

**Pull request merged**